test/NuGet.CommandLine.Staging.UnitTests/StagingCommandUnitTests.cs

```diff
@@ -39,7 +41,7 @@ public void StageCommand_WhenSourceNotProvidedErrorIsShown()
         public void StageCommand_WhenApiKeyNotProvidedErrorIsShown()
         {
             // Act
-            string[] args = { "stage", "-create", "abc", "-Source", "http://api.nuget.org/v3/index.json" };
+            string[] args = { "stage", "create", "abc", "-Source", "http://api.nuget.org/v3/index.json" };

```

Comment:
[Should this also be `serverV3.Uri + "index.json"`?](https://github.com/NuGet/NuGet.Staging/pull/68/files#diff-c32ca6667e824a67153d9f7d9379bbacR73)